isis/tests/Fixtures.cpp

```diff
@@ -65,12 +65,11 @@ namespace Isis {
 
 
   void ThreeImageNetwork::TearDown() {
+    delete cubeList;
+    delete network;
+  
     delete cube1;
     delete cube2;
     delete cube3;
```

Comment:
Is there a good reason for not using smart pointers for these?

Reply:
Realistically we shouldn't have to. I think these tests are exposing a memory problem in ISIS (I'm thinking right now something involving cubes being double closed left and right, causing cleanup to be called multiple times) that smart pointers wouldn't really fix. I'm gonna create a Linux build and try to replicate it. Maybe something about Macs build tools is more robust to these sorts of things?